<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-3P6EQHG" failures="1" tests="6" name="Test" time="39.269" errors="0" timestamp="2022-09-06T15:43:42 EDT">
  <testcase classname="Testcase.dsalgotest" name="verifyhomepage" time="0.713">
    <failure type="java.lang.AssertionError" message="expected [NumpyNinja1] but found [NumpyNinja]">
      <![CDATA[java.lang.AssertionError: expected [NumpyNinja1] but found [NumpyNinja]
at dsalgo.homepage.Test1(homepage.java:72)
at Testcase.dsalgotest.verifyhomepage(dsalgotest.java:50)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verifyhomepage -->
  <testcase classname="Testcase.dsalgotest" name="verifylogin" time="0.711"/>
  <testcase classname="Testcase.dsalgotest" name="verifylogin" time="0.967"/>
  <testcase classname="Testcase.dsalgotest" name="verifylogin" time="1.17"/>
  <testcase classname="Testcase.dsalgotest" name="reg" time="9.248"/>
  <testcase classname="Testcase.dsalgotest" name="arr" time="16.73"/>
</testsuite> <!-- Test -->
